CONTRIBUTING.md

```diff
@@ -220,10 +220,16 @@ You can lint a single file (but this will _not_ exclude legacy errors):
 - Style rules are (mostly) defined by `src/uncrustify.cfg` which tries to match
   the [style-guide]. To use the Nvim `gq` command with `uncrustify`:
   ```
-  if !empty(findfile('src/.uncrustify', ';'))
-    setlocal formatprg=uncrustify\ -q\ -c\ src/uncrustify.cfg\ --replace\ --no-backup
+  if !empty(findfile('src/uncrustify.cfg', ';'))
+    setlocal formatprg=uncrustify\ -q\ -l\ C\ -c\ src/uncrustify.cfg\ --no-backup
```

Comment:
is `-l C` actually needed?

Reply:
Yeah it was quite the debugging journey to figure out what's wrong with it. Try to remove that flag and also remove the `-q` then you'll be able to see the following:

> If reading from stdin, you should specify the language using -l
or specify a filename using --assume for automatic language detection.

Reply:
It is needed when reading from stdin (which formatprg does).